.github/cluster.yaml

```diff
@@ -10,8 +10,6 @@ iam:
 addons:
 - name: aws-ebs-csi-driver
   serviceAccountRoleARN: "arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy"
-  wellKnownPolicies:
```

Comment:
The reason we needed this setting was to be able to create PVs backed by EBS in AWS .... I just wonder do we know what will be the default storeage class without this config option ? Will we be still able to create RWmany PVs ?

Reply:
If you notice, we assign the [AmazonEBSCSIDriverPolicy](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonEBSCSIDriverPolicy.html) serviceAccount either way, so we should be fine.